plugins/network-elements/nuage-vsp/pom.xml

```diff
@@ -28,6 +28,13 @@
     <version>4.6.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.1</version>
```

Comment:
This dependency has been added since one of the methods of our private client returns an object of type Pair which is inside the commons-lang3 library